README.md

```diff
@@ -1,12 +1,50 @@
-<h1 align=center>ML Enabler v3</h1>
+<h1 align=center>ML Enabler (MLE) v3</h1>
 
-<p align=center>Frictonless TF based model storage, inferencing & discovery</p>
+<p align=center>Frictonless model storage, inferencing & discovery</p>
 
 ## Background
 
-Machine Learning has proven to be very successful to make mapping fast with high quality. With a diverse set of models and tools, it is hard to integrate them to existing tools like Tasking Manager and iD. ML Enabler is a service that provides better integration during model development as well integrating model inferences to mapping tools.
+Machine Learning has proven to be very successful to make mapping fast with high quality. With a diverse set of models and tools, it is hard to integrate them to existing tools like Tasking Manager and iD. ML Enabler (MLE) is a service that provides better integration during model development as well integrating model inferences to mapping tools.
 
-ML Enabler was initially developed by Development Seed in partnership with Humanitarian OpenStreetMap Team.
+MLE was initially developed by Development Seed in partnership with Humanitarian OpenStreetMap Team.
+
+## Features of MLE
+
+### Model Registry
+
+MLE serves as a registry of public and private machine learning models. It can track models from disparate organizations all in one place and take advantage of all of them.
+
+### On-Demand Prediction
+
+MLE makes it easy to spin up AWS infrastructure to run your model along with all necessary resources. MLE UI allows you to upload new models & generate and preview predictions, all you need is a TMS end point and an AOI.
+
+### Classification, Detection & Segmentation model support
+
+MLE supports all three categories of CV problems i.e classification, detection & segmentation.
+
+### Feedback loop
+
+MLE supports a feedback loop about predictions from within the interface. Users can tag a tile as valid or invalid. Predictions tagged as valid switch to green, predictions tagged as invalid switch to white, and predictions that haven’t been manually validated stay red.
+MLE can then convert these validated predictions back into labelled training data matched up with imagery to allow users to easily re-train a new model with the validated model predictions.
+
+## Current Limitations
+
+### Infrastructure
+
+MLE uses AWS CloudFormation and is tightly integrated with AWS infrastructure at the moment. We can add support for other cloud providers or private infrastructure based on request.
```

Comment:
Let's remove ```
We can add support for other cloud providers or private infrastructure based on request.
```
This would be a massive rewrite at this point